source/guide_prosody.rst

```diff
@@ -0,0 +1,309 @@
+
+Now we copy the configuration and plugin files into reasonable locations, download the community plugins and create the prosody-data and http_upload directory:
+
+::
+
+ [isabell@stardust ~]$ cp -R $PROSODY_PATH/conf/ ~/etc/prosody
+ [isabell@stardust ~]$ cp -R $PROSODY_PATH/plugins/ ~/var/prosody/plugins
+ [isabell@stardust ~]$ mkdir -p ~/var/prosody/data/http_upload
+ [isabell@stardust ~]$ hg clone https://hg.prosody.im/prosody-modules/ ~/var/prosody/community-plugins
+ [isabell@stardust ~]$
+
+Then there are many settings which should be edited accordingly in ``~/etc/prosody/prosody.cfg.lua``:
+
+.. code-block:: lua
+
+ admins = { "isabell@uber.space" }
+ plugin_paths = {"/home/isabell/var/prosody/community-plugins/"}
+ -- uncomment/add "proxy65" and "http_upload" in modules_enabled
+ allow_registration = false
+ c2s_require_encryption = true
+ c2s_ports = { CLIENTPORT }
+ s2s_require_encryption = true
+ s2s_secure_auth = true
+ s2s_ports = { SERVERPORT }
+ authentication = "internal_hashed"
+ pidfile = "/home/isabell/var/prosody/prosody.pid";
+ daemonize= false;
+ storage = "sql"
+ sql = { 
+   driver = "MySQL", 
+   database = "isabell_prosody", 
+   username = "isabell", 
+   password = "<Your Password>", 
+   host = "localhost" 
+ }
+ log = { info = "*console" }
+ certificates = "/home/isabell/etc/certificates"
+ https_certificate = "../certificates/stardust.space.crt"
+ http_ports = {}
+ https_ports = { FILEUPLOADPORT }
+ proxy65_ports = { FILETRANSFERPORT }
+ proxy65_address = "stardust.space"
+ proxy65_acl = {"stardust.space"}
+ VirtualHost "stardust.space" 
+ Component "groupchat.stardust.space" "muc"
+   modules_enabled = { "muc_mam" } 
+
+.. warning:: Replace the placeholders ``CLIENTPORT``, ``SERVERPORT``, ``FILEUPLOADPORT`` and ``FILETRANSFERPORT`` with the above obtained ports, adapt the domain-names, sql settings (inclusive username and password) and paths! Don't delete, obmit or change the ordering of the entries, otherwise some default ports could be spammed. Also don't active modules which including module ``http`` without changing ``http_ports`` and ``https_ports`` . Last but not least be warned that spamming the default ports which could already be in use can lead to fork-spam issues! So be careful and watch your configuration twice and look into the prodoy logs afterwards to verify whats going on after starting prosody!  
+
+Finishing Installation
+======================
+
+supervisord
+-----------
+
+Place the file ``prosody.ini`` in ``~/etc/services.d/`` and adapt it accordingly:
+
+.. code-block:: console
+
+ [program:prosody]
+ command=/bin/bash -c "source /home/isabell/.bash_profile && prosody"
+ autostart=yes
+ autorestart=yes
+ startretries=1
+ stopasgroup=true
+
+
+First-Start
+-----------
+
+Reread:
+
+::
+
+ [isabell@stardust ~]$ supervisorctl reread
+ prosody: available
+ [isabell@stardust ~]$
+
+And then update and start your new service:
+
+::
```

Comment:
Replace this with the new `.. include:: includes/supervisord.rst`, please :)

Reply:
Done!